### ISSUE TITLE
service worker generation?

### ISSUE THREAD

**Issue by oL-web** [lifenautjoe/webpack-starter-basic#3]
create-react-app has it, google's abandoned web starter kit has it.

i really like this starter kit so adding new features would be pretty neat. i really like how react can turn a basic web app into a PWA without doing anything.

btw i'd be nice to see webpack-dev-server host a server on the local network the same way create-react-app does. i just added my ip to the --host flag to webpack-dev-server in npm start but it would be nice to see this starter kit do it automatically out of the box.

**Comment by lifenautjoe** [comment]
Hey Michal, 

Thanks for the suggestion :)

Do you mean generating a service worker for offline caching? 

On exposing the dev server on the local area network, I'll add an npm script to do it.

**Comment by oL-web** [comment]
Yeah, offline caching and the ability to add the website to the phone's homescreen. create-react-app has service worker generation that does everything automatically, the developer only needs to worry about meeting the PWA checklist criteria(such as https) for the web app to have all these magical functionalities.
